src/start.ts

```diff
@@ -12,7 +12,7 @@ import { ctx } from "./ctx";
 import { logger } from "./logger";
 
 logger.debug("Creating client...");
-const client = new Client({ intents: ["Guilds", "GuildMembers"] });
+const client = new Client({ intents: ["Guilds", "GuildMembers", "GuildPresences"] });
```

Comment:
Should only include this intent if `logActivity` is true.

Reply:
true